### PR TITLE
BOAC-4417: YCBM appointments now show start and end time of appt

### PR DESCRIPTION
**Pull request by cesarvh**
https://jira.ets.berkeley.edu/jira/browse/BOAC-4417

### CODE REVIEW

File: src/components/student/profile/AcademicTimelineTable.vue [hunk 1]
```diff
@@ -265,6 +265,12 @@
                     :sr-prefix="message.type === 'appointment' ? 'Appointment date' : 'Created on'"
                     :include-time-of-day="(message.createdAt.length > 10) && (message.type !== 'appointment')"
                   />
+                  <div
+                    v-if="message.createdBy === 'YCBM' && message.endsAt"
+                    :id="`expanded-${message.type}-${message.id}-appt-time-range`"
+                  >
+                    {{ formatSameDayDate(message) }}
```
Comment (pauline2k) [inline src/components/student/profile/AcademicTimelineTable.vue:272]: @cesarvh Should this be calling your new `getSameDayDate` method?

File: boac/merged/advising_appointment.py [hunk 2]
```diff
@@ -202,6 +202,7 @@ def appointment_to_compatible_json(appointment, topics=(), attachments=None, eve
         'createdBy': created_by,
         'deptCode': appointment.get('dept_code'),
         'details': appointment.get('details'),
+        'endsAt': appointment.get('ends_at').isoformat() if created_by == 'YCBM' else None,
```
Comment (pauline2k) [inline boac/merged/advising_appointment.py:205]: @cesarvh Looks like this is causing a lot of test failures with `'NoneType' object has no attribute 'isoformat'`. I think you'll want to include another condition here along the lines of `if 'ends_at' in appointment`.

**Comment by cesarvh** [comment]
@pauline2k  Hmm.. I don't why why this says I closed the PR after I sent in the changes, but now it wont let me make a new one or reopen it. Are you able to?

**Comment by pauline2k** [comment]
Github voodoo. Looks like it's behaving itself now…

**Pull request merged**